app/views/user_mailer/registration_confirmation.text.slim

```diff
@@ -0,0 +1,5 @@
+p Hi #{@user.name},
+
+p Thanks for registering! To confirm your registration click the URL below.
+
+href="http://localhost:3000/sign_up/confirm_email?token=#{@user.confirm_token}"
```

Comment:
In the link there should be ENV variable with URL for the frontend.Locally this ENV will be localhost, but on production and test server different.

Reply:
Fixed.